addons/web/static/src/views/kanban/kanban_dashboard.scss

```diff
@@ -105,11 +105,10 @@
                     div[class*="col-"] + div[class*="col-"] {
                         border-left: 1px solid map-get($grays, '300');
                     }
-
-                    // Default options box
-                    div.text-end {
-                        text-align: left; // :/
```

Comment:
Why **was** needed and not anymore?  RTL text? Could you check?

Reply:
@stefanorigano It's the only kanban card that had this custom class. I think it's because in this module the card was coded differently than the others ( and that's why it's the only place broken). There was no "dropdown-item" class like in all the others modules and they seems to have customized it this way. I don't think it would be relevant to make it text-end like it was.
I see no signs of rtl text needed?

Reply:
Well, I had a look at the commits history and it's not clear to me neither :/
Look ok though